tox.ini

```diff
@@ -1,13 +1,22 @@
 [tox]
 # note that tox expects interpreters to be found at C:\PythonXY,
 # with XY being python version ("27" or "34") for instance
-envlist = py26, py27, py32, py33, py34, docs
+envlist = py{27,34}-pyqt4, py34-pyqt5, py{26,27,32,33,34}-pyside, docs
+
+
 [testenv]
-deps=pytest 
-    pyside    
+deps=pytest
+    py{26,27,32,33,34}-pyside: pyside
 commands=py.test {posargs}
```

Comment:
This actually runs the system-wide pytest for me - I think `{envpython} -m py.test` would be better.